Modules/arraymodule.c

```diff
@@ -2106,6 +2106,19 @@ array__array_reconstructor_impl(PyObject *module, PyTypeObject *arraytype,
     return result;
 }
 
+static PyObject*
+get_array_reconstructor(void)
+{
+    _Py_IDENTIFIER(_array_reconstructor);
+    PyObject *array_module = PyImport_ImportModule("array");
+    if (array_module == NULL)
+        return NULL;
+    PyObject *meth = _PyObject_GetAttrId(array_module,
```

Comment:
Actually we can use `PyObject_GetAttrString()`. In any case this is called only once. No need to keep `_array_reconstructo` in a list of identifiers.